src/Views/ShortcutsView.vala

```diff
@@ -124,4 +96,17 @@ public class ShortcutOverlay.ShortcutsView : Gtk.Grid {
         add (new Gtk.Separator (Gtk.Orientation.VERTICAL));
         add (column_end);
     }
+
+    private void add_shortcut_entries (Gee.ArrayList<ShortcutEntry> entries, Gtk.Grid column) {
+        for (int a = 0; a < entries.size; a++) {
```

Comment:
Could you change `a` to `i`? It's generally a more recognizable pattern in for loops.